cfme/tests/webui/test_splitter.py

```diff
@@ -51,16 +50,16 @@
 )
```

Comment:
REQUIRED: you forgot to change location above

Reply:
I'm not sure I follow?

Reply:
psav on L43 in your branch, there's a use of location within `unblock=lambda location: ...`

Reply:
not any more